cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_search_facets_and_results.html

```diff
@@ -149,23 +137,8 @@ <h3 class="h2">Filter results by</h3>
             </div>
             {% endcall %}
         {% endif %}
-        {% if facets.topic %}
-            {% if 'topic' in page.results.expanded_facets %}
-                {% set is_expanded = true %}
-            {% else %}
-                {% set is_expanded = false %}
-            {% endif %}
-            {% set expandable_settings = {
-                'label': 'Topic',
```

Comment:
@mrclay Is this logic no longer needed? I thought it was used to make sure the topics are expanded when a facet is selected. Or are topics now expanded no matter what?

Reply:
Never mind. Found it up on line 94.